src/Agent.Listener/Configuration/ConfigurationManager.cs

```diff
@@ -409,14 +401,19 @@ public AgentSettings LoadSettings()
                     await agentSvr.ConnectAsync(conn);
                     Trace.Info("Connect complete.");
 
+                    Type agentType = command.DeploymentAgent ? typeof(DeploymentAgentConfiguration) : typeof(AutomationAgentConfiguration);
```

Comment:
should we put this information to .agent? so we don't have to prompt during un-configure